README.md

```diff
@@ -29,6 +29,8 @@ This project uses GitHub [milestones](https://github.com/kubernetes-incubator/no
 
 ## Command line interface
 
+One way to run stand-alone is by running a Docker container where node-feature-discovery is set as entry point.
```

Comment:
Hmm, what usage are you thinking here? First of all, in nfd images node-feature-discovery is already set as the default entry point. Of course, you can run the nfd image directly on docker, but then the labeling fails because the k8s APIs of the cluster are not available. I think this should be mentioned, at least, probably with a recommendation to use --no-publish to avoid the related errors.